Lib/test/test_ntpath.py

```diff
@@ -374,6 +374,12 @@ def test_realpath_basic(self):
         self.assertPathEqual(ntpath.realpath(os.fsencode(ABSTFN + "1")),
                          os.fsencode(ABSTFN))
 
+        # gh-88013: call ntpath.realpath with binary drive name may raise a
+        # TypeError. The drive should not exist to reproduce the bug.
+        drives = {f"{chr(x)}:\\" for x in range(65, 91)} - set(os.listdrives())
```

Comment:
```suggestion
        drives = {f"{c}:\\" for c in string.ascii_uppercase} - set(os.listdrives())
```

(will also need an `import string` at the top)